explore/src/main/scala/explore/Explore.scala

```diff
@@ -17,22 +15,19 @@ import gem.Observation
 @JSExportTopLevel("Explore")
 object ExploreMain extends AppMain {
 
-  override def rootComponent(
-    WithModelCtx: AppRoot.Component[IO, explore.AppContextIO, RootModel]
-  ): VdomElement =
-    WithModelCtx { viewCtx =>
-      val routing = new Routing(viewCtx) // !!! This creates a new router on each render.
+  override def rootComponent(viewCtx: ViewCtxIO[RootModel]): VdomElement = {
+    val routing = new Routing(viewCtx) // !!! This creates a new router on each render.
```

Comment:
Is there a big drawback on creating a new router each time?

Reply:
Yes, huge. This actually remounts the whole application on a state change, since it creates a new `Router` component. This won't be acceptable behavior moving forward.

For the moment, the solution I found is to tweak the `Router` to allow properties going through it (PR submitted to `scalajs-react`), which allows us to use the same component when state changes.